src/BuildScriptGenerator/StagingRuntimeConstants.cs

```diff
@@ -7,5 +7,12 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Common
     public static class StagingRuntimeConstants
     {
         public static readonly List<string> DotnetcoreStagingRuntimeVersions = new List<string> { };
+        public static readonly List<string> PythonStagingRuntimeVersions = new List<string> { };
+        public static readonly List<string> NodeStagingRuntimeVersions = new List<string> { };
+        public static readonly List<string> JavaStagingRuntimeVersions = new List<string> { };
+        public static readonly List<string> PhpStagingRuntimeVersions = new List<string> { };
+        public static readonly List<string> HugoStagingRuntimeVersions = new List<string> { };
+        public static readonly List<string> RubyStagingRuntimeVersions = new List<string> { };
+        public static readonly List<string> GolangStagingRuntimeVersions = new List<string> { };
     }
```

Comment:
Could you please clarify why we need empty lists here?

Reply:
Just figured these empty lists were propogated by the following constants defined in constants.yaml file. 

```
constants:
    dotnetcore-staging-runtime-versions:
      -
    python-staging-runtime-versions:
      -
    node-staging-runtime-versions:
      -
    java-staging-runtime-versions:
      -
    php-staging-runtime-versions:
      -
    hugo-staging-runtime-versions:
      -
    ruby-staging-runtime-versions:
      -
    golang-staging-runtime-versions:
      -    
```